bower.json

```diff
@@ -7,7 +7,9 @@
   "private": false,
   "dependencies": {
     "jquery": "latest",
-    "dropzone": "git://github.com/cyrilkyburz/dropzone"
+    "dropzone": "git://github.com/cyrilkyburz/dropzone",
+    "jasmine-core": "jasmine#^2.4.1",
```

Comment:
wrong dependencies,we can now use official dropzone & we need jasmine only in development